api/src/main/scala/pl/touk/nussknacker/engine/api/component/ComponentType.scala

```diff
@@ -49,6 +50,7 @@ object ComponentType extends Enumeration {
     case _: CustomNodeData => Some(CustomNode)
     case _: Enricher => Some(Enricher)
     case _: Processor => Some(Processor)
+    case _: BranchEndData => Some(BranchEnd)
```

Comment:
Test is failing because I've added this new type - not sure then if this is really needed? 

Is `BranchEnd` a component?

Reply:
No, `BranchEnd` is not an component. It is special node that make special topologies of our graphs possible (mainly joins) to be written as a tree structure.

Reply:
Removed 👍